charts/core/traefik/Chart.yaml

```diff
@@ -22,7 +22,7 @@ sources:
 - https://github.com/traefik/traefik-helm-chart
 - https://traefik.io/
 type: application
-version: 11.0.13
+version: 11.0.14
```

Comment:
As requested, please bump the *minor* version, as this is a feature addition not a patch.